source/includes/api/performance/_self_claims.md

```diff
@@ -0,0 +1,176 @@
+## Self Claims
+
+### Get Self Claims
+
+Endpoint to get self claims against a specific self claim promotion. Claims can be scoped down via passed in parameters. If no url parameters are provided then all claims against the requested promotion are returned. Results are paginated at 100 records per page.
+
+
+#### HTTP Request
+
+`GET /api/v2/performance/promotions/{promotion_id}/self_claims`
+
+> Header:
+
+``` http
+GET /api/v2/performance/promotions/1/self_claims HTTP/1.1
+Authorization: Token token=xxx
+Content-Type: application/json
+```
+
+> Response:
+
+``` http
+HTTP/1.1 200 Ok
+Content-Type: application/json
+
+{
+    "self_claims": [
+        {
+            "id": 4099,
+            "sale_date": "29/01/2019",
+            "product_or_activity_ref": "p1",
+            "quantity": 150,
+            "points_value": 150,
+            "user_id": 505,
+            "status": "pending",
+            "declined_reason": null,
+            "created_at": "30/01/2019",
+            "updated_at": "07/01/2021",
```

Comment:
ISO format for dates? Don't know what we use elsewhere in the API

Reply:
I replicated the allocated claims endpoint format here - am checking wtih Jules to see what she wants